src/Controls/src/Core/Compatibility/Handlers/ListView/iOS/ListViewRenderer.cs

```diff
@@ -690,6 +690,9 @@ void ReloadData()
 				Control.ReloadData();
 			else
 				PerformWithoutAnimation(() => { Control.ReloadData(); });
+
+			if (Control.NumberOfRowsInSection(0) > 0)
```

Comment:
The test **GroupListViewHeaderIndexOutOfRangeTest** is crashing on iOS:
<img width="912" alt="image" src="https://github.com/user-attachments/assets/23781f03-9e0e-47d7-b2ca-3cc8ab5a4e2d">

`The app was expected to be running still, investigate as possible crash`